timy/__init__.py

```diff
@@ -25,36 +25,52 @@ def output(ident, text):
 
 class Timer(object):
 
-    def __init__(self, ident=timy_config.DEFAULT_IDENT):
+    def __init__(self, ident=timy_config.DEFAULT_IDENT, include_sleep=True):
         self.ident = ident
         self.start = 0
+        self.end = 0
+        self.include_sleep = include_sleep
 
     def __enter__(self):
-        self.start = time.time()
+        if self.include_sleep:
+            self.start = time.perf_counter()
+        else:
+            self.start = time.process_time()
         return self
 
     def __exit__(self, type, value, traceback):
         output(self.ident, '{:f}'.format(self.elapsed))
 
     @property
     def elapsed(self):
-        return time.time() - self.start
+        if self.include_sleep:
+            self.end = time.perf_counter()
+        else:
+            self.end = time.process_time()
+        return self.end - self.start
 
     def track(self, name='track'):
         output(self.ident, '({}) {:f}'.format(name, self.elapsed))
 
 
-def timer(ident=timy_config.DEFAULT_IDENT, loops=1):
+def timer(ident=timy_config.DEFAULT_IDENT, loops=1, include_sleep=True):
```

Comment:
We can apply the same idea here, having a variable holding the correct function depending on the parameter. And then replace both ifs with it.